ResultRandomizer

arichards611 opened the pull request:
Randomizes results in hipchat.py so issue #1 is addressed and changed search text in dank.json

arichards611 committed:
ResultRandomizer

Randomizes results in hipchat.py so issue #1 is addressed and changed search text in dank.json

gelstudios commented:
the journey of a thousand miles begins with a single pull request grasshopper... :+1:

arichards611 committed:
Changed random method

Changed randomization method from randint to random.choice
Changed import to include all of "random"
Removed unused/commented code
Fixed white space

gelstudios commented:
looks great, thanks!
